nemo_text_processing/text_normalization/en/taggers/telephone.py

```diff
@@ -98,7 +103,14 @@ def __init__(self, deterministic: bool = True):
             NEMO_DIGIT ** (1, 3), digit + pynini.closure(pynutil.insert(" ") + digit)
         ).optimize()
         ip_graph = digit_to_str_graph + (pynini.cross(".", " dot ") + digit_to_str_graph) ** 3
-        graph |= pynutil.insert("number_part: \"") + ip_graph.optimize() + pynutil.insert("\"")
+        graph |= (
+            pynini.closure(
+                pynutil.insert("country_code: \"") + ip_prompts + delete_extra_space + pynutil.insert("\""), 0, 1
```

Comment:
what's the expected behavior if ip/telephone prompts are missing? default to serial class?

Reply:
please take a look at test cases, it should be fine if they are missing. then it would be old behavior.